### ISSUE TITLE
Some elements are fully black and unreadable

### ISSUE THREAD

**Issue by aaronhowser1** [brandon3055/ProjectIntelligence#16]
![](https://cdn.discordapp.com/attachments/613467918176616458/637850226316673029/2019-10-26_23.09.07.png)

Something like #11, but Smoothfont is fully updated. Would it still be on their end?

**Comment by FoxMcloud5655** [comment]
Have you tried without Smoothfont?  If so and it's not there, then it would probably be their fault still.

**Comment by aaronhowser1** [comment]
![image](https://user-images.githubusercontent.com/15698722/67629495-9dc66400-f844-11e9-86a5-c8fcc1328fca.png)
I removed it, and it worked. BUT, it still showed the exact same as the original image and the text loaded in after it. Could that be an issue still?

**Comment by FoxMcloud5655** [comment]
I'm not versed enough in Project Intelligence to give you a proper answer to that, but I would assume that it might be something brandon can fix, even if it isn't just a compatibility patch.

**Comment by brandon3055** [comment]
What version of SmoothFont do you have? This should be fixed in v2.1+

**Comment by aaronhowser1** [comment]
SmoothFont-mc1.12.2-2.1.1

**Comment by brandon3055** [comment]
I can't seem to reproduce this issue. Not with the latest SmoothFont. Its possible there is a new trigger causing the issue. This used to occur whenever a blue slime from tinkers rendered on the screen. But that's not the case any more.

**Comment by aaronhowser1** [comment]
![image](https://user-images.githubusercontent.com/15698722/67629711-0a8f2d80-f848-11e9-95d8-fe7d03cec363.png)
Right after making a new world

![image](https://user-images.githubusercontent.com/15698722/67629715-1a0e7680-f848-11e9-9d2a-7af339b42ab1.png)
After closing the GUI and spawning a bunch of Blue Slimes.

This is using Smoothfont 2.1.1 and ProjectIntelligence 1.0.8.25.
![image](https://user-images.githubusercontent.com/15698722/67629724-4629f780-f848-11e9-8595-792f95c5a09f.png)

**Comment by brandon3055** [comment]
I really don't know... I am running the same versions as you. 
![](https://ss.brandon3055.com/8db91)
Is this a public mod pack? Did you download your mods directly from curse?

**Comment by aaronhowser1** [comment]
Yes, the latest version of [FTB Academy](https://www.curseforge.com/minecraft/modpacks/ftb-academy), downloaded through Twitch.

**Comment by Azriel-Legnasia** [comment]
Same issue for as well, with the same modpack, although I did load it to MultiMC.
It will occasionally show the proper element colors as well before the error kicks in again after a few seconds

**Comment by Bineary** [comment]
For anyone else experiencing this on the FTB Academy pack here's a workaround:
Press the "home" key to bring up SmoothFont's menu and click the button in the top left of the screen to disable SmoothFont. Project Intelligence seems to work fine when that button shows "Mod Disabled" or "Optimize Only" and if you want to enable it again after using Project Intelligence for what you needed you can do that the same way.

_Smooth Font set to "Optimize Only"_
![SmoothFontOptimizeOnly](https://user-images.githubusercontent.com/4999459/88469987-2d2ca080-cec5-11ea-8fca-0cc33bd96904.png)


I'm experiencing the same issue though I have not seen any slimes yet in my world. I have however looked through the entire "Materials and You" book from Tinker's Construct multiple times and I see that the book contains a tooltip for blue slime crystals on one of the pages. I'm not sure if that would be enough to trigger the slime related issue. I didn't see this noted by others but it also affects any tooltips in Project Intelligence. 

_Smooth Font set to "Mod Enabled"_
![SmoothFontOn](https://user-images.githubusercontent.com/4999459/88470006-59e0b800-cec5-11ea-9aaa-511424c13090.png)

_"Materials and You" book showing blue slime crystal tooltip_
![BlueSlimeCrystalTinkersBook](https://user-images.githubusercontent.com/4999459/88470017-6cf38800-cec5-11ea-9fb3-72b82fe51b16.png)


I'm running the latest version of the FTB Academy modpack (1.2.0) via the FTB Launcher, I have not altered the pack. These are the versions shown in the mod list for the relevant mods (I think BrandonsCore is relevant to PI?):
 
Smooth Font Core | Version: mc1.12.2-2.1.1
Smooth Font | Version: mc1.12.2-2.1.1
Brandon's Core | Version: 1.12.2-2.4.18.210
Project Intelligence | Version: 1.12.2-1.0.9.28

**Comment by Bineary** [comment]
Update to the above, after following the link in the first post to issue #11 and then finding the link to the [issue submitted on Smooth Font's tracker](https://www.curseforge.com/minecraft/mc-mods/smooth-font/issues/18) I noticed the Smooth Font dev stated it was fixed in version 2.1.2 whereas the FTB Academy pack is currently using version 2.1.1. I manually updated Smooth Font to 2.1.2 from Curse and tested it again. With Smooth Font 2.1.2 installed Project Intelligence seems to be displaying fine:
![2020-07-25_22 35 26](https://user-images.githubusercontent.com/4999459/88470366-53ecd600-cec9-11ea-9d23-e322c9412da8.png)

[I've submitted an issue to the FTB Academy tracker for this](https://github.com/FTBTeam/FTB-Academy/issues/317), hopefully the version of Smooth Font included in the mod will be updated.
